erpnext/patches/v8_0/update_student_groups_from_student_batches.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.utils.rename_field import *
+from frappe.model.mapper import get_mapped_doc
+
+
+def execute():
+	if frappe.db.table_exists("Student Batch"):
+		student_batches = frappe.db.sql('''select name as student_group_name, student_batch_name as batch,
+			program, academic_year, academic_term from `tabStudent Batch`''', as_dict=1)
+
+		for student_batch in student_batches:
+			if frappe.db.exists("Student Group", student_batch.get("student_group_name")):
+				student_group = frappe.get_doc("Student Group", student_batch.get("student_group_name"))
+
+				if frappe.db.table_exists("Student Batch Student"):
+					current_student_list = frappe.db.sql('''select student from `tabStudent Group Student`
+						where parent=%s''', (student_group.name), as_list=1)
+					current_student_list = [d for student in current_student_list for d in student]
```

Comment:
What is this?
You can achieve this using `frappe.db.sql_list`. Atleast you can use `[d[0] for student in current_student_list]`